### DISCUSSION TITLE
utiliziting Wyze motion detection

### DISCUSSION THREAD

**Discussion by Anc0dia** [blakeblackshear/frigate#5428] [Ideas]
I noticed that the Wyze cameras will put a green box around detected objects/motion.   Since Frigate uses alot of cpu power processing motion detection perhaps Frigate could have a processing mode where it only looks for those green boxes in the stream and when found sends the image in the box off to the image inference processor  This way all the motion detection work is done on the camera.

Just a thought.

**Comment by blakeblackshear** [comment]
It will take a similar amount of resources to analyze the frames and look for green boxes.
